site/content/en/blog/releases/1.21.0-rc.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.21.0 - Release Candidate"
+linkTitle: "1.21.0-rc"
+date: "2022-02-08"
+---
+This is the 1.21.0-rc release of Agones.
+
+This is a big bug fix release!
+
+Not only did we fix a tonne of flaky tests (which will make contributing easier), but also solved lots of tricky bugs in the process. This includes fixes to the alpha Fleet Autoscaling custom syncing, as well as quality of life improvements where we now validate `metadata` patches on `GameServerAllocation`.
```

Comment:
nit: s/tonne/ton

Reply:
Is it an imperial ton or metric ton (tonne) though, and in the latter case should we use the SI unit of megagrams instead? :P

Reply:
https://en.wikipedia.org/wiki/Tonne 😄